PicPi/FrameAPI.swift

```diff
@@ -37,8 +43,8 @@ enum FrameAPIErrors: Error {
 protocol FrameAPI {
   /// Upload the selected photo with filename
   func upload(photoData: Data, filename: String) -> Promise<Void>
-  /// Get the list of URLs of the thumbnails stored on the Pi
-  func retrieveThumbnails() -> Promise<[URL]>
+  /// Get the list of photos and their URLs of thumbnails or full images stored on the Pi
+  func retrievePhotos() -> Promise<[PhotoData]>
```

Comment:
wdyt of changing it to retrievePhotosInfo() ?
that will make it more readable :)
retrievePhotos()  will give the impression of retrieving the actual photo

Reply:
sgtm 😁